README.md

```diff
@@ -27,36 +27,35 @@ with the following goals in mind:
 Please checkout the *Projects* tab for information about roadmap and current progress.
 
 # Setting up Minotaur on Windows
-These will describe the basic steps for setting up an environment on Windows.
-Things may vary from PC to PC, though.
-
-### Installing Qt
-1. Download the Qt community edition and install a version at least 5.6
-2. I recommend 5.6 or 5.7 since the later versions take up much more space
-3. After installation completes set the `QT_DIR` environment variable, which would look
-something like `C:\Qt\%VERSION%\bin`.
-4. Add the some variable to `PATH`
-
-### Installing Python
-1. Download and install Python 3.6
-2. You might want to add the python `include` and `lib` directories to your `PATH`
-
-### Running with CLion
-I recommend using CLion, since it's more self contained and does work for you.
-
-1. Download and install CLion, which is free for students
-2. If you don't have the Microsoft Visual Studio compiler on your machine
-you can either install MVS (yikes!) or follow these steps
-https://www.jetbrains.com/help/clion/quick-tutorial-on-configuring-clion-on-windows.html
-3. Make sure you install a relatively recent compiler version
-
-## Building Minotaur
-1. `git clone https://github.com/uwnrg/minotaur-cpp.git`
-2. Open the project in CLion by clicking `Import from sources`
-3. CLion will attempt to detect a compiler
-4. CLion will attempt to configure the cmake project
-5. If that goes well, you can build the project
-6. If you get any errors, ping me `@jeffniu22` on Slack
+These will describe the exact steps for setting up an environment
+on a clean installation of Windows 10.
```

Comment:
only windows 10?

Reply:
I've only tried on Windows 10, and I won't be presumptuous about other versions unless someone tries and succeeds.

Reply:
I see, mention that it's only tested on Windows 10 instead